Sofascore/Sofascore/ViewControllers/MainViewController.swift

```diff
@@ -0,0 +1,114 @@
+import UIKit
+import SofaAcademic
+import SnapKit
+
+class MainViewController: UIViewController {
+    
+    private let customHeader: HeaderCustomView = .init()
+    private let tabView: CustomTabView = .init()
+    private let containerView: UIView = .init()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        addViews()
+        setupConstraints()
+        styleViews()
+        tabView.selectTab(for: Sport(rawValue: UserDefaults.standard.integer(forKey: "sport")) ?? .football)
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        customHeader.isHidden = false
+    }
+    
+    override func viewWillDisappear(_ animated: Bool) {
+        customHeader.isHidden = true
+    }
+    
+    private func showViewController(for sport: Sport) {
+        let eventsViewController = EventsViewController(sport: sport)
+        add(eventsViewController, containerView)
+        eventsViewController.view.snp.makeConstraints {
+            $0.edges.equalToSuperview()
+        }
+    }
+}
+
+//MARK: UIViewController
+extension UIViewController {
+    
+    func add(_ child: UIViewController,_ view: UIView) {
+        let animation = CATransition()
+        animation.duration = 0.3
+        animation.type = .push
+        animation.subtype = .fromRight
```

Comment:
Ovo je bas jako opcionalno, mozda bolje reci za one koji zele znati vise. Pisem ti samo jer je zadaca bas jako dobra i realno nemas bas nesto za popravljati pa ako se zelis pozabaviti i s ovim. 
Sloziti neku logiku ovdje da imas fromRight i fromLeft ovisno o polozaju. Ja bi ti savjetovala ako ces to raditi da ostavis ovu ekstenziju za pocetak ovdje (i zanemaris moj komentar povise) i roknes sve u ovu metodu pa ces lako poslije preraspodijeliti kod. Takodjer, slobodno me pingaj za to